src/components/link.rs

```diff
@@ -67,7 +68,18 @@ pub fn link<T>(props: &LinkProperties<T>) -> Html
 where
     T: Target + 'static,
 {
-    let router = use_router::<T>().expect("Need Router or Nested component");
+    info!("Render Link {:?}", props.to);
```

Comment:
I'd prefer not to have into logging in a case like that. I guess it can be lowered to `trace` or at least `debug`.